lib/TransferControllers/AsyncCopyControllers/AsyncCopyController.cs

```diff
@@ -331,7 +331,7 @@ private void SetHasWorkAfterStatusChanged()
                 await this.DoFetchSourceAttributesAsync();
             }
 #if EXPECT_INTERNAL_WRAPPEDSTORAGEEXCEPTION
```

Comment:
As discussion offline, we might create a new work item to merge the "if define branches" as .net core has changed their behavior.